packages/react-sdk/src/db/style.ts

```diff
@@ -0,0 +1,49 @@
+import type { StyleProperty, StyleValue } from "@webstudio-is/css-data";
+import { SharedStyleValue } from "@webstudio-is/css-data";
+import { type ComponentName, getComponentMeta } from "../components";
+import { z } from "zod";
+
+export type PresetStyleItem = {
+  component: ComponentName;
+  property: StyleProperty;
+  value: StyleValue;
+};
+
+// @todo can't figure out how to make component and property to be enum
+export const zPresetStyleItem = z.object({
+  component: z.string(),
+  property: z.string(),
+  value: SharedStyleValue,
+}) as z.ZodType<PresetStyleItem>;
+
+export const zPresetStyle = z.array(zPresetStyleItem);
```

Comment:
we have been using the convention for same-named type and zod schema, so before switching it, we need to discuss it and figure out if its worth it switching everywhere

Reply:
Typescript hover is broken and shows any when same name is used

Reply:
that's news to me, needs investigation, that wasn't a problem before

Reply:
tested, no problem there, its likely your ts was still parsing or something, try restarting ts in your editor

Reply:
Reproduced this in the same module where both defined. Values are overriden by types. So when hover value I get any.